c#/SqliteDbManager.cs

```diff
@@ -1,27 +1,23 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Data.Common;
-using System.Data.SQLite;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using Microsoft.Data.Sqlite;
 
-namespace Backend
+namespace Backend;
+
+public class SqliteDbManager : IDbManager
 {
-    public class SqliteDbManager : IDbManager
+    public DbConnection getConnection()
```

Comment:
nit: getConnection should probably be GetConnection to follow the format used everywhere else